lib/services/local/users.go

```diff
@@ -419,6 +425,156 @@ func (s *IdentityService) DeleteSignupToken(token string) error {
 	return trace.Wrap(err)
 }
 
+// This is hardcoded in the U2F library
+const u2fChallengeTimeout = 5 * time.Minute
```

Comment:
lets move this to defaults:

https://github.com/gravitational/teleport/blob/master/lib/defaults/defaults.go